src/util/MongoBinary.js

```diff
@@ -155,6 +155,10 @@ export default class MongoBinary {
     if (systemBinary) {
       binaryPath = await this.getSystemPath(systemBinary);
       if (binaryPath) {
+        if(~binaryPath.indexOf(' ')) {
```

Comment:
Please add `// eslint-disable-line o-bitwise`

And run lint before committing:
`npm run lint`